magento/magento2
# Fix - Next Page button triggered when filtering Customer grid

### Description
I do change in grid.phtml. There is a missing button type. If we did not put button type then button work as a submit button.

### Fixed Issues (if relevant)
1. magento/magento2#17789: Next Page button triggered when filtering Customer grid

### Manual testing scenarios
1. Go to "Sales -> Orders"
2. "Create new order"
3. Customer grid is displayed
4. Focus on any column filter (i.e. text box in email column)
5. Press ENTER to trigger the filter

### Expected result

1. Filter's AJAX call is triggered
2. Results don't change because the filter input was empty
3. Grid is still showing the same page.

### Contribution checklist
